mailgun/route_handlers.py

```diff
@@ -43,14 +43,14 @@ def handle_mailing_list_email_route(request):
 
     member_addresses = [m['address'] for m in ml.members]
     bounce_back_email_template = None
-    if ml.access_level == MailingList.ACCESS_LEVEL_MEMBERS and sender not in member_addresses:
+    if ml.get_listserv_access_level == MailingList.ACCESS_LEVEL_MEMBERS and sender not in member_addresses:
```

Comment:
You'll need to make the property change recommended below in order for this to work.